apps/application-system/form/src/index.html

```diff
@@ -38,6 +38,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
 
+    <script
+      async
+      defer
+      data-domain="island.is/umsoknir"
```

Comment:
Interesting. Is it an issue if this is hard coded and running on all environments?

It looks like this is what [service portal](https://github.com/island-is/island.is/blob/99c6b7c33cad398ce6f70bd7deee4e66d89c923d/apps/service-portal/src/index.html#L44) is doing. They're also using the outbound-links version of the script if we want to be consistent.